src/stylus/settings/_theme.styl

```diff
@@ -73,7 +73,7 @@ $material-light := {
   picker: {
     body: #fff,
     clock: $grey.lighten-2,
-    title: $theme.primary
+    title: 'inherit'
```

Comment:
This commit breaks the default picker colors, here's the playground for test, compare with dev branch:
```vue
<template>
  <v-app id="inspire" :dark="dark">
    <v-content>
      <v-checkbox v-model="dark" label="Dark"></v-checkbox>
      <v-layout row><v-date-picker v-for="i in 3" style="width: 290px" :light="i === 2" :dark="i === 3"></v-date-picker></v-layout>
    </v-content>
  </v-app>
</template>

<script>
export default {
  data: () => ({ dark: false })
}
</script>
```
The main problem is that dark and light themes use different colors for picker title (primary for light theme, gray for dark theme), otherwise it could be easily fixed in `picker.js` by changing line 32 from `return this.headerColor || this.color` to for example `return this.headerColor || this.color || 'primary'`

Moreover after removing `background: $material.picker.title` from `_pickers.styl` the `$material-*.picker.title` variable is not used any longer and can be removed